### PR TITLE
Added Pilagá Language

### PR DESCRIPTION
**Pull request by bjpardal**
We developed and added a new language, Pilagá, to the existing bunch of languages.
The Pilagás or Pilagaes (in Pilagá language: pit'laxá) are an indigenous people of the Guaicurú group, who live in the center of the province of Formosa in Argentina. They have their own language, that is why we decided to do this project as a thesis for the Systems Engineering degree, to preserve the language and prevent technology from being a barrier in the daily communication of the community.

### Portrait Layout:
|Capital letters|Lower case letters|Predictive|
|----------|----------|----------|
|||()|

### CODE REVIEW

File: addons/languages/pilaga/apk/build.gradle [hunk 1]
```diff
@@ -0,0 +1,7 @@
+ext.shouldBePublished = true
```
Comment (menny) [inline addons/languages/pilaga/apk/build.gradle:1]: For the first PR, we'll need this to be set to `false`. After a successful manual release, we can switch to `true`

File: addons/languages/pilaga/apk/release/output-metadata.json
Comment (menny) [inline addons/languages/pilaga/apk/release/output-metadata.json:1]: This file needs to be deleted. I'm not sure where it came from.

File: buildSrc/src/main/java/Parser.java [hunk 2]
```diff
@@ -168,8 +168,8 @@ private char fixup(int intChar) {
         switch (intChar) {
             case 8217: // back-tick `
                 return '\'';
-            case 8221: // fancy open-quotes ”
-            case 8220: // fancy close-quotes “
+            case 8221: // fancy open-quotes �
+            case 8220: // fancy close-quotes �
```
Comment (menny) [inline buildSrc/src/main/java/Parser.java:172]: I'm not sure what has happen here, please revert this change.

**Review by menny — CHANGES_REQUESTED** [review]
Thank you for this contribution. I have a few requests for changes.

File: addons/languages/pilaga/apk/build.gradle (hunk 1 above)
Comment (bjpardal) [reply addons/languages/pilaga/apk/build.gradle]: Done. Thanks!

File: addons/languages/pilaga/apk/release/output-metadata.json
Comment (bjpardal) [reply addons/languages/pilaga/apk/release/output-metadata.json:1]: Done. Thanks!

File: buildSrc/src/main/java/Parser.java (hunk 2 above)
Comment (bjpardal) [reply buildSrc/src/main/java/Parser.java]: Done. Thanks!

File: buildSrc/src/main/java/Parser.java [hunk 3]
```diff
@@ -168,9 +168,6 @@ private char fixup(int intChar) {
         switch (intChar) {
             case 8217: // back-tick `
                 return '\'';
-            case 8221: // fancy open-quotes ”
```
Comment (menny) [inline buildSrc/src/main/java/Parser.java:171]: case 8221: // fancy open-quotes ”
            case 8220: // fancy close-quotes “
                return '\"';

**Review by menny — CHANGES_REQUESTED** [review]
The file `Parser.java` should not change at all.

File: addons/languages/pilaga/pack/build.gradle [hunk 4]
```diff
@@ -0,0 +1,5 @@
+ext.status_icon_text = 'pi'
+ext.dictionaryInputPossibleCharacters = null//.toCharArray()
+ext.dictionaryInputAdditionalInnerCharacters = null//.toCharArray()
+
+apply from: "$rootDir/addons/gradle/language_pack_lib.gradle"
```
Comment (menny) [inline addons/languages/pilaga/pack/build.gradle:5]: missing new-line at the end of the file

**Review by menny — CHANGES_REQUESTED** [review]

File: buildSrc/src/main/java/Parser.java (hunk 3 above)
Comment (bjpardal) [reply buildSrc/src/main/java/Parser.java:171]: Sorry for this, we thought that we had added those lines unintentionally, so we deleted. And now, we realized that there was a character that it was wrong.

File: addons/languages/pilaga/pack/build.gradle (hunk 4 above)
Comment (bjpardal) [reply addons/languages/pilaga/pack/build.gradle]: Done

**Review by menny — APPROVED** [review]
thank you

**Comment by menny** [comment]
Have you generated new IDs?
https://github.com/AnySoftKeyboard/AnySoftKeyboard/actions/runs/7293919709/job/19901199550#step:6:1

**Review by menny — APPROVED** [review]

**Comment by menny** [comment]
It is now available on the Play Store: https://play.google.com/store/apps/details?id=com.anysoftkeyboard.languagepack.pilaga
